spec/services/search/base_spec.rb

```diff
@@ -17,6 +17,14 @@
       described_class.index(document_id, id: document_id)
       expect(described_class.find_document(document_id)).not_to be_nil
     end
+
+    it "sets last_indexed_at field" do
+      Timecop.freeze(Time.current) do
+        described_class.index(document_id, id: document_id)
+        last_indexed_at = described_class.find_document(document_id).dig("_source", "last_indexed_at")
+        expect(Time.parse(last_indexed_at).to_i).to eq(Time.current.to_i)
```

Comment:
here we should use `Time.zone.parse` as `Time.parse` can technically fail in some timezones (hopefully one day all time methods will be timezone aware as time without a timezone doesn't really make sense for a computer :D, like text without an encoding)